pyartcd/pyartcd/locks.py

```diff
@@ -25,7 +25,7 @@ class Lock(enum.Enum):
     Lock.OLM_BUNDLE: {
         'retry_count': 36000,
         'retry_delay_min': 0.1,
-        'lock_timeout': 60 * 60 * 2,  # 2 hours
+        'lock_timeout': 60 * 30 # 30 minutes
```

Comment:
Why changing this? I think we still want to keep the pre-defined locks lifetime, or we'll be releasing resources before we actually finished using them.